INSTALL.md

```diff
@@ -83,12 +83,14 @@ your `.zshrc`.
     git clone https://github.com/zsh-users/zsh-syntax-highlighting.git ${ZSH_CUSTOM:-~/.oh-my-zsh/custom}/plugins/zsh-syntax-highlighting
     ```
 
-2. Activate the plugin in `~/.zshrc`:
+2. Activate the plugin **at the bottom** of `~/.zshrc`:
 
     ```zsh
     plugins=( [plugins...] zsh-syntax-highlighting)
```

Comment:
This doesn't work for me. I had to add spaces like this or it errored out:
```sh
plugins=( [ plugins... ] zsh-syntax-highlighting )
```

Reply:
Those brackets are not meant to be entered literally.  Wherever you entered that command literally with spaces, you just disabled all your other plugins.  Change the command to `plugins+=( zsh-syntax-highlighting )`.

This change can and should be made in INSTALL.md as well.